flowtorch/parameters/dense_autoregressive.py

```diff
@@ -161,6 +165,7 @@ def _forward(
         else:
             h = x
 
+        # Why not using regular sequential?
```

Comment:
This is another good point! I'm not sure why the following lines just don't evaluate a `torch.nn.Sequential`. It may have had something to do with when you have skip connections perhaps?